ipaserver/install/server/replicainstall.py

```diff
@@ -738,6 +775,7 @@ def install_check(installer):
                          ldap_uri=ldapuri)
     remote_api.finalize()
     installer._remote_api = remote_api
+
```

Comment:
It was not removed but rather moved to `install_check()` in https://github.com/freeipa/freeipa/pull/445/files as this is a DL-specific check from now on.